applications/nrf5340_audio/broadcast_sink/overlay-broadcast_sink.conf

```diff
@@ -6,7 +6,6 @@
 
 CONFIG_TRANSPORT_BIS=y
 CONFIG_AUDIO_DEV=1
-CONFIG_BT_DEVICE_NAME="NRF5340_BIS_HEADSET"
```

Comment:
Discussing this. @Frodevan: Could we potentially set the name for headset #2 to something else in the test system?